python/ray/services.py

```diff
@@ -789,3 +814,21 @@ def start_ray_head(address_info=None,
                              start_workers_from_local_scheduler=start_workers_from_local_scheduler,
                              num_cpus=num_cpus,
                              num_gpus=num_gpus)
+
+def new_log_files(name):
+  """Generate partially randomized filenames for log files.
+
+  Args:
+    name (string): descriptive string for this log file.
```

Comment:
`str` instead of `string`. Description sentence should stop with a capitalized letter.